__tests__/app.test.js

```diff
@@ -0,0 +1,147 @@
+const request = require('supertest');
+const app = require('../app'); 
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data/index')
+const db = require('../db/connection')
+const endpointsJSON = require ('../endpoints.json')
+
+
+
+beforeEach(() => {
+return seed(testData)
+
+})
+
+afterAll(() => {
+  return db.end()
+})
+
+describe('GET /api/topics', () => {
+
+
+
+  it('should respond with status 404 for an invalid endpoint', () => {
+    return request(app)
+      .get('/api/invalid')
+      .expect(404);
+  });
+
+  it('should respond with an array of objects that is not empty', () => {
+    return request(app)
+      .get('/api/topics')
+      .expect(200)
+      .then((res)=>{
+        expect(res.body.topics.length).not.toBe(0)
```

Comment:
We can be more specific here, look in the database test data and check what the length should